CMakeLists.txt

```diff
@@ -19,8 +19,20 @@ if(NOT CMAKE_BUILD_TYPE)
     set(CMAKE_BUILD_TYPE "Debug" CACHE STRING "Enable debug or release builds" FORCE)
 endif()
 
-if (NOT DEFINED USE_LOG4CPP)
-  option(USE_LOG4CPP "Enable LOG4CPP" ON)
+option(USE_SPDLOG "Enable SPDLOG" OFF)
+option(USE_LOG4CPP "Enable LOG4CPP" OFF)
+option(USE_INTERNAL_LOGGER "Enable concord-bft's logger" OFF)
+
+if (USE_SPDLOG)
+    message("-- spdlog enabled")
+    set(USE_INTERNAL_LOGGER OFF)
```

Comment:
it is a cmake option, if it is provided by the user it will be ON, in this case I set the other 2 logger options to off. The log4cplus remains the default one. We propagate this option as a define to the build in [logging/CMakeLists.txt:45](https://github.com/vmware/concord-bft/blob/a5f98ea568fe8560eab9fcca64a0ff0b3d859ec0/logging/CMakeLists.txt#L45)